torch/_inductor/fx_passes/pad_mm.py

```diff
@@ -184,6 +184,15 @@ def is_mm_compute_bound(M: int, K: int, N: int, dtype: torch.dtype) -> bool:
         return False
     arithmetic_intensity = (M * N * K) / denominator
 
+    # we have experienced some large perf hits in this case, even in bandwidth bound regimes
+    if (
+        dtype is torch.bfloat16
+        and K > M
+        and K > N
```

Comment:
This is pretty permissive, does that seem ok?  Any downsides?

Reply:
Happy to tighten it. The main reason we skipped bandwidth bound mms was bc of kv cache - you would have a bunch of mms that were all bandwidth bound of slightly different shape and it really increased compile time. Not an issue here..

We've seen pretty large perf issues with bfloat16 unaligned so I think as is is fine.

Reply:
Cool, makes sense to me!